### ISSUE TITLE
FileNotFoundError: [Errno 2] No such file or directory:  'C:\\[...].jpg'

### ISSUE THREAD

**Issue by hkarrson** [pywavefront/PyWavefront#101]
Hey! I'm writing a game engine that needs to be capable of loading obj and mtl files.
However, the wavefront/obj format isn't completely standardized, so different programs handle
things in different ways. I found that often mtl files will use absolute paths that the original
creator used while creating the mesh, rather than relative paths. The problem with this is that
the end user may not have the full original path the creator used, but they should have a relative
path containing the exact filename. I propose that this library could allow users to create
some sort of callback so they can handle paths used in files themselves and return the actual
path that should be returned. So I could have a function that says "def handleFilePath(path, type): return newPath" and I also think it would be good to have an option in the load function to just do a subdir search. 

To clarify, here was the problem I faced: 

FileNotFoundError: [Errno 2] No such file or directory: 'C:\\DocumentsandSettings\\AGH\\MyDocuments\\3dsMax\\sceneassets\\images\\FlammableBarrelTexture.jpg'

And here is the relavent mtl "code":

	map_Ka C:\Documents and Settings\AGH\My Documents\3dsMax\sceneassets\images\Flammable Barrel Texture.jpg
	map_Kd C:\Documents and Settings\AGH\My Documents\3dsMax\sceneassets\images\Flammable Barrel Texture.jpg

This is clearly not the proper filepath, since I don't have that filepath, but the problem is
that in certain programs like 3D Viewer, it's still textured, which is why this should be solved
in the code and not in the file.

**Comment by einarf** [comment]
I suggest adding a `find` method in `Texture`: https://github.com/pywavefront/PyWavefront/blob/master/pywavefront/texture.py

It would check if `path` actually exists and fall back to using `glob` in `pathlib` to do a quick subdirectory scan.

**Comment by hkarrson** [comment]
I agree completely, but I also think it may be important to allow developers to handle this in there own way as well.

**Comment by einarf** [comment]
Agreed. It could just be there as an optional tool. Right now the library just present the user with the raw `path` and `name` of the texture and are left to their own devices anyway regarding how this information should be used.

**Comment by hkarrson** [comment]
PR: https://github.com/pywavefront/PyWavefront/pull/102
